examples/complete/main.tf

```diff
@@ -94,7 +94,7 @@ module "postgresql_db" {
         attributes = [
           {
             "name" : "endpointType",
-            "value" : "private"
+            "value" : "private,public"
```

Comment:
The only idea was to include a example to have both public & private.  Since we are including a VPE will make this as private.  But we should think of adding this as part of some example.